cms/templates/js/unit-access-editor.underscore

```diff
@@ -0,0 +1,110 @@
+        return;
+    }
+    if (partition['scheme'] == 'enrollment_track') {
+        var tracks = [];
+        var trackSelected = false;
+        partition['groups'].forEach(function (group) {
+            if (group['selected'] === true) {
+                trackSelected = true;
+            }
+        })
+        if (partition['groups'].length <= 1 && trackSelected === false) {
+            return;
+        }
+    }
+    selectablePartitions.push(partition);
+})
+selectablePartitions.forEach(function (partition) {
+    var schemeId = partition['id'];
+    var groups = partition['groups'];
+    groups.forEach(function (group) {
+        var groupId = group['id'];
+        var groupName = group['name'];
+        var fullId = schemeId.toString() + '-' + groupId.toString();
+        groupIdToName[fullId] = groupName;
+    })
+})
+%>
+<form>
+<h3 class="modal-section-title">
+    <% if (xblockInfo.isVertical()) { %>
+        <%- gettext('Unit Visibility') %>
+    <% } else { %>
+        <%- gettext('Section Visibility') %>
+    <% } %>
+</h3>
+<div class="modal-section-content staff-lock">
+    <ul class="list-fields list-input">
+        <li class="field field-checkbox checkbox-cosmetic">
+            <input type="checkbox" id="staff_lock" name="staff_lock" class="input input-checkbox" />
+            <label for="staff_lock" class="label">
+              <span class="icon fa fa-check-square-o input-checkbox-checked" aria-hidden="true"></span>
+              <span class="icon fa fa-square-o input-checkbox-unchecked" aria-hidden="true"></span>
+              <%- gettext('Hide from learners') %>
+            </label>
+
+            <% if (hasExplicitStaffLock && !ancestorLocked) { %>
+                <p class="tip tip-warning">
+                  <% if (xblockInfo.isVertical()) { %>
+                    <%- gettext('If the unit was previously published and released to learners, any changes you made to the unit when it was hidden will now be visible to learners.') %>
+                  <% } else { %>
+                    <% var message = gettext('If you make this %(xblockType)s visible to learners, learners will be able to see its content after the release date has passed and you have published the unit. Only units that are explicitly hidden from learners will remain hidden after you clear this option for the %(xblockType)s.') %>
+                    <%- interpolate(message, { xblockType: xblockType }, true) %>
+                  <% } %>
+                </p>
+            <% } %>
+        </li>
+    </ul>
+</div>
+<br>
+
+<% if (xblockInfo.isVertical()) { %>
+    <h3 class="modal-section-title">
+        <%- gettext('Unit Access') %>
+    </h3>
+    <div class="modal-section-content access-change">
+        <ul class="list-fields list-input">
+            <li>
+            <%- gettext('Restrict access to:') %>
+            <select class="user-partition-select">
+                <option selected="selected" value="none">All Learners and Staff</option>
```

Comment:
Remember to mark all user-visible strings for translation, and unless a variable is expected to contain HTML, you should not be using <%=

Also don't forget i18n and safe template practices.
http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/internationalization/i18n.html

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html